code/src/components/MovieList.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState} from 'react';
+import { MovieThumb } from './MovieThumb';
+
+import "./movieList.css";
+
+export const MovieList = () => {
+    const MOVIES_URL = 'https://api.themoviedb.org/3/movie/popular?api_key=0cb608022c0a0fbef75881f68133d694&language=en-US&page=1';
+    const [movies, setMovies] = useState([]);
+
+    console.log('Render');
+
+    useEffect(() => {
+        fetch(MOVIES_URL)
+        .then((response) => response.json())
+        .then((json) => setMovies(json.results));
+    }, []);
+
+    return (
+        <section className="movie-list">
+            {movies.map((movie) => ( 
+            <MovieThumb 
+                key={movie.id}
+                name={movie.title} 
+                background_image={movie.poster_path}
+                release={movie.release_date}
+                id={movie.id}
+                />
```

Comment:
Good that you're passing specific parts of the data from the api fetch into the MovieThumb. I found the `{...movie}` solution a good one so that you can pass all the data for each object into the MovieThumb.

Reply:
Also, indentation could be improved over here